packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_custom_icons.tsx

```diff
@@ -948,3 +948,27 @@ export const CWEnvelop = ({
     </svg>
   );
 };
+
+export const CWShareIcon = () => {
```

Comment:
we dont use this approach for basic icons. Please use "export" icon from phosphor https://phosphoricons.com/?q=%22export%22

(look at `cw_icon_lookup.ts`)